templates/build/small-card-loop.build.hbs

```diff
@@ -10,7 +10,7 @@
         \{{/if}}
       </div>
     </div>
-    <div class="small-card-list-text-wrapper">
+    <div class="small-card-list-text-wrapper \{{#if bookmarksEnabled}} bookmark-padding \{{/if}}">
```

Comment:
I would recommend calling this `has-bookmarks`, @tonytlwu do you agree?

Reply:
@tonytlwu ^

Reply:
I agree

Reply:
@tonytlwu @squallstar Just to confirm in this case I will have to change the name from bookmarksEnabled to has-bookmarks though all the code. 
<img width="1137" alt="bookmarks demo" src="https://user-images.githubusercontent.com/52824207/65219574-82d43780-dac1-11e9-9d3c-5893c1dabbf6.png">

Reply:
@maksim-an-upplabs nope, what we meant is to change your `bookmark-padding` class to be named `has-bookmarks`.

Reply:
@squallstar Fixed